js/app.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+const all_animals = [];
+
+function Animal (horn){
+    this.image_url = horn.image_url;
+    this.title = horn.title;
+    this.description = horn.description;
+    this.keyword = horn.keyword;
+    this.horns = horn.horns;
+  all_animals.push(this);
+}
+
+Animal.prototype.render = function(){
+  const photo_section_html = $('#photo-template').html()
+
+  $('main').append('<section id="clone"></section>');
+  $('#clone').html(photo_section_html);
+  $('#clone').find('h2').text(this.name);
+  $('#clone').find('img').attr('src', this.image_url);
+  $('#clone').find('p').text(this.title).text(this.description);
+  $('#clone').attr('id', '');

```

Comment:
You may want to use some classes or ids to more easily target these sections when filtering.